include/memory_debugging.h

```diff
@@ -41,11 +41,27 @@
 #include <valgrind/memcheck.h>
 
 #define MEM_UNDEFINED(a, len) VALGRIND_MAKE_MEM_UNDEFINED(a, len)
-#define MEM_DEFINED_IF_ADDRESSABLE(a, len) \
-  VALGRIND_MAKE_MEM_DEFINED_IF_ADDRESSABLE(a, len)
+// #define MEM_DEFINED_IF_ADDRESSABLE(a, len) \
```

Comment:
nit: why do you comment these?

Reply:
See the `#pragma GCC poison` below and the comment there: it's not called in the sources